bins/packages/runc/runc.sh

```diff
@@ -0,0 +1,73 @@
+RUNC_VERSION="1.0.0-rc9"
+RUNC_CHECKSUM="e88bcb1a33e7ff0bfea495f7263826c2"
+RUNC_LINK="https://github.com/opencontainers/runc/archive/v${RUNC_VERSION}.tar.gz"
+
+dependencies_runc() {
+    apt-get install -y btrfs-tools libseccomp-dev build-essential pkg-config
+
+    if [ -z $GOPATH ]; then
+        if command -v go > /dev/null; then
+            export GOPATH=$(go env GOPATH)
+        else
+            curl -L https://dl.google.com/go/go1.13.1.linux-amd64.tar.gz > /tmp/go1.13.1.linux-amd64.tar.gz
```

Comment:
Can we use the latest version of go instead?

Reply:
This code is not used when running on github action, we install go (1.15) first, then execute this.
This part of code is executed only when doing standalone and is mainly for debug purpose. We can update it, sure, but it doesn't have any impact on resulting flist.

Reply:
Got it.